cypress-tests/cypress/support/pageBuilder/createMenu.ts

```diff
@@ -0,0 +1,45 @@
+import { gqlClient } from "../utils";
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
```

Comment:
Check the same things reported in other commands.

Reply:
I am having many issues getting this up to par with the other commands.
Since the old command was working properly I have deleted this filed and moved the old pbCreateMenu.js -> pbCreateMenu.ts